From 129fbcb37df0fae85c80701271161a6438756cae Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Mon, 6 Sep 2010 11:22:23 +0200
Subject: [PATCH 12/16] Do not check for unused `opie' library

* configure.ac: Remove opie AC_CHECK_LIB check.
(cherry picked from commit db4d2c949b277c155ab36296e01b48276dc6277e)
---
 configure.ac |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 90b0c29..0ef46ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -140,7 +140,6 @@ gl_INIT
 AC_CHECK_LIB(nsl, inet_ntoa)
 AC_CHECK_LIB(resolv, gethostname)
 AC_CHECK_LIB(socket, getpeername)
-AC_CHECK_LIB(opie, opiegenerator)
 
 # See if a termcap library is available (under one of several names)
 IU_LIB_TERMCAP
-- 
1.7.2.3

